concent_api/generate-requirements-lock.sh

```diff
@@ -2,9 +2,21 @@
 
 if [[ "$VIRTUAL_ENV" != "" ]]
 then
-    echo "--find-links https://builds.golem.network/simple/pyelliptic" > ${BASH_SOURCE%/*}/requirements.lock
-    pip freeze >> ${BASH_SOURCE%/*}/requirements.lock
-    echo "Done"
+    if [ "$#" -ne 1 ]
```

Comment:
This statement is not needed here. You are using the first argument, so you do not need to check the number of parameters.

Reply:
there could be no parameters ;)

Reply:
This case can be covered by checking if the file is empty, your decision.

Reply:
I left it as it is for greater warnings readability.

Reply:
Please, modify it for readability.
```bash
[[ "$#" != 1 ]]
```

Reply:
Done